src/NUnitFramework/testdata/nunit.testdata.csproj

```diff
@@ -1,30 +1,18 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net35;net40;net46;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>$(NUnitRuntimeFrameworks)</TargetFrameworks>
     <RootNamespace>NUnit.TestData</RootNamespace>
-
-    <!-- Microsoft.Bcl.Build workaround https://github.com/nunit/nunit/issues/2663 -->
-    <SkipValidatePackageReferences>true</SkipValidatePackageReferences>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
-    <TargetFrameworks>net35;net40;net46;netcoreapp3.1</TargetFrameworks>
-  </PropertyGroup>
-
-  <PropertyGroup Condition="'$(OS)' != 'Windows_NT'">
-    <TargetFrameworks>net35;net46;netcoreapp3.1</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0" />
     <PackageReference Include="jnm2.ReferenceAssemblies.net35" Version="1.0.1" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net40'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net40' OR '$(TargetFramework)' == 'net45'">
```

Comment:
I'm not familiar with this change, but I'm assuming it fixes something :)

Reply:
It fixes consistency. nunit.framework.csproj was compiled with net35;net40;net45.
Other assemblies had various other version. They are now all aligned. The condition includes some Microsoft.Bcl packages for functionality that was added to the framework in 4.6